pkg/META

```diff
@@ -98,8 +98,8 @@ package "syntax" (
     description = "Syntax extension: predefined commonly use syntaxes"
     version = "[distributed with Eliom]"
 
-    requires(syntax, preprocessor) = "js_of_ocaml.syntax,js_of_ocaml-camlp4.deriving,lwt_camlp4"
-    requires(syntax, toploop) = "js_of_ocaml.syntax,js_of_ocaml-camlp4.deriving,lwt_camlp4"
+    requires(syntax, preprocessor) = "js_of_ocaml-camlp4,js_of_ocaml-camlp4.deriving,lwt_camlp4"
+    requires(syntax, toploop) = "js_of_ocaml-camlp4,js_of_ocaml-camlp4.deriving,lwt_camlp4"
     archive(syntax, preprocessor) = "-ignore dummy"
```

Comment:
Maybe iit would be time to retire the camlp4 extension instead ?

Reply:
Strategically it might be wise to wait for (or rather implement) PPX support for PGOCaml. Then we could migrate the entirety of the ocsigen projects to PPX.